raiden/transfer/channel.py

```diff
@@ -482,6 +494,10 @@ def is_valid_lock_expired(
         msg = 'Invalid LockExpired message. Same lockhash handled twice.'
         result = (False, msg, None)
 
+    elif lock_registered_on_chain:
+        msg = 'Invalid LockExpired message. Secret registered on-chain.'
```

Comment:
I think we can clarify this, the secret being registered on-chain doesn't imply the lock was unlocked. The lock is unlocked iff the block number in which the secret was registered is before the lock expiration.

So I think `Invalid LockExpired mesage. Lock was unlocked on-chain` is a bit more unambiguous